Assets/Countly/Plugins/CountlySDK/Helpers/RequestCountlyHelper.cs

```diff
@@ -109,18 +123,21 @@ private async Task<CountlyResponse> ProcessRequest(CountlyRequestModel model)
             bool shouldPost = _config.IsForcedHttpPostEnabled() || model.RequestData.Length > 2000;
 
 #if UNITY_WEBGL
-            // There is not HTTP GET for webGL. We always do a HTTP POST
+            // There is not HTTP GET for WebGL. We always do a HTTP POST
+            Log.Debug($"[RequestCountlyHelper] ProcessRequest, Calling StartProcessRequestRoutine for WebGL");
             return await StartProcessRequestRoutine(_countlyUtils.ServerInputUrl, model.RequestData);
 #else
             if (shouldPost) {
+                Log.Debug($"[RequestCountlyHelper] ProcessRequest, Using HTTP POST");
```

Comment:
cant see the shouldPost is legit or not, double check pls

Reply:
`bool shouldPost = _config.IsForcedHttpPostEnabled() || model.RequestData.Length > 2000;`
lgtm. also it was working fine, what concerned you?